src/bin/nydus-image/main.rs

```diff
@@ -544,8 +554,13 @@ fn init_log(matches: &ArgMatches) -> Result<()> {
 
     // Safe to unwrap because it has a default value and possible values are defined.
     let level = matches.value_of("log-level").unwrap().parse().unwrap();
+    let rotate_size = matches
+        .value_of("log-rotate-size")
```

Comment:
Better to rename it to `log-rotation-size`

Reply:
Ok, understand.